src/benchmarks-julia.jl

```diff
@@ -1,62 +1,97 @@
-#!/usr/bin/env julia
+options = Base.JLOptions()
+
+@assert VERSION >= v"1.0.2"
+@assert options.can_inline == 1
+@assert options.check_bounds == 2
+@assert options.opt_level == 3
 
 using RigidBodyDynamics
-using StaticArrays
+using Random
+using LinearAlgebra
 
 include("models.jl")
 
-function benchmark_julia_rnea(model, log_filename)
-    robot = parse_urdf(Float64, PATH * "$model.urdf")
-    if model != "lwr"
-        for joint in out_joints(root_body(robot), robot)
-            floatingjoint = Joint(string(joint), frame_before(joint), frame_after(joint), QuaternionFloating{Float64}())
-            replace_joint!(robot, joint, floatingjoint)
-        end
+function write_times(log_filename, times)
+    file = open(log_filename, "w") do io
+        println.(Ref(io), times[2 : end]) # skip the first sample, which includes compilation time
     end
+    return nothing
+end
 
-    states = Array{MechanismState}(undef, NBT)
-    for i = 1:NBT
-        states[i] = MechanismState(robot)
-        rand!(states[i])
+function benchmark_julia_rnea(model, log_filename)
+    robot = parse_urdf(PATH * "$model.urdf", floating=model != "lwr")
```

Comment:
Use the (relatively new) simpler API for creating a `Mechanism` with a floating joint.